wbooks/src/app/components/Navigator/index.tsx

```diff
@@ -4,15 +4,34 @@ import { createStackNavigator } from '@react-navigation/stack';
 import Routes from '@constants/routes';
 import Library from '@screens/Library';
 import DetailBook from '@screens/DetailBook';
+import Header from '@components/Header';
+import BackIcon from '@components/BackIcon';
+import TextHeader from '@components/TextHeader';
 
 const Stack = createStackNavigator();
 
 function AppNavigator() {
   return (
     <NavigationContainer>
       <Stack.Navigator>
-        <Stack.Screen name={Routes.Home} component={Library} options={{ title: 'Library' }} />
-        <Stack.Screen name={Routes.Detail} component={DetailBook} />
+        <Stack.Screen
+          name={Routes.Home}
+          component={Library}
+          options={{
+            headerBackground: () => <Header />,
+            headerTitle: () => <TextHeader title="Library" />
+          }}
+        />
+        <Stack.Screen
+          name={Routes.Detail}
+          component={DetailBook}
+          options={{
+            headerBackground: () => <Header />,
+            headerTitle: () => <TextHeader title="Book Detail" />,
+            headerBackImage: () => <BackIcon />,
```

Comment:
You would receive tintColor so you might want to support it in case in a future you need it
```suggestion
            headerBackImage: BackIcon,
```

Reply:
Awesome! I didn't know this